src/test/java/com/unzer/payment/service/UrlUtilTest.java

```diff
@@ -49,11 +49,11 @@ public void testGetUrlForPaymentTypeInvoiceSecured() throws HttpCommunicationExc
     @Test
     public void testGetUrlForPaymentTypeInstallmentSecured() throws HttpCommunicationException {
         String minimalRestUrl = "https://api.unzer.com/v1/types";
-        String maximumRestUrl = "https://api.unzer.com/v1/types/invoice-secured";
+        String maximumRestUrl = "https://api.unzer.com/v1/types/installment-secured";
 
         Unzer unzer = getUnzer();
         UrlUtil urlUtil = new UrlUtil(unzer.getEndPoint());
-        InstallmentSecuredRatePlan installmentSecuredRatePlan = createPaymentTypeInstallmentSecured();
+        InstallmentSecuredRatePlan installmentSecuredRatePlan = createPaymentTypeInstallmentSecuredRatePlan();
```

Comment:
How does ist work if u fetch another resource then a payment type? Does this function effect fetching other resources then payment types? "urlUtil.getHttpGetUrl(resource.id))) "